safe_transaction_service/contracts/tests/test_tx_decoder.py

```diff
@@ -392,3 +392,33 @@ def test_db_tx_decoder(self):
         self.assertEqual(fn_name, "buyDroid")
         self.assertEqual(arguments, {"numberOfDroids": "4", "droidId": "10"})
         self.assertIn((contract.address,), DbTxDecoder.cache_abis_by_address)
+
+        #  Test fallback calls
```

Comment:
Please do it in a different function. We should split this test also, I don't know why we put everything together